src/Sulu/Bundle/MediaBundle/Content/Types/ImageMapContentType.php

```diff
@@ -351,12 +351,18 @@ function(ContentTypeInterface $contentType, $property) {
 
     public function getContentData(PropertyInterface $property)
     {
-        return $this->prepareData(
+        $data = $this->prepareData(
             $property,
             function(ContentTypeInterface $contentType, $property) {
                 return $contentType->getContentData($property);
             }
         );
+
+        if (!($data['image'] ?? null)) {
```

Comment:
Doesn't `!isset($data['image'])` work as well? This line looks a bit confusing and unusual to me.